lib/widgets/enable_backup_dialog.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:breez/theme_data.dart' as theme;
+import 'package:breez/bloc/backup/backup_bloc.dart';
+
+class EnableBackupDialog extends StatefulWidget {
+  final BuildContext context;
+  final BackupBloc backupBloc;
+
+  EnableBackupDialog(this.context, this.backupBloc);
+
+  @override
+  EnableBackupDialogState createState() {
+    return new EnableBackupDialogState();
+  }
+}
+
+class EnableBackupDialogState extends State<EnableBackupDialog> {
+  bool _isChecked = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return showEnableBackupDialog();
```

Comment:
Dialog shouldn't be shown from the build function because every trigger to build again will prompt with another dialog... 
We can just leave a function here: 
"createEnableBackupDialog" which returns an AlertDialog and then when needed just call:
showDialog(createEnableBackupDialog())

Reply:
Looking again, actually this is OK since it only returns a widget. Taking back the feedback. sorry.